### DISCUSSION TITLE
Metacat

### DISCUSSION THREAD

**Discussion by pacian** [NCEAS/metacatui#2110] [Q&A]
Dear Community,

I am from the Biomedical Community and was asked to review metacat as a potential solution to store metadata for large datasets that reside elsewhere, i.e., EMBL. Can the solution be used ONLY as a metadata catalog solution without storing datasets?

I appreciate any help you can give me.

Richard

**Comment by mbjones** [comment]
Hi @pacian Thanks for your question. Yes, `metacat` (https://github.com/NCEAS/metacat) is an open source metadata and data repository catalog that can be used to store, query display, and serve a wide variety of metadata standards. Metacat supports several programming APIs, including the [DataONE API](https://search.dataone.org/api). `metacatui` (https://nceas.github.com/metacatui) is a client-side javascript application that presents a nice metadata search, display, and editing. Pretty much any XML-based or JSON-based, or structured standard is in theory supportable, but it requires some work to map a new standard into the search system. We support many common metadata standards, such as ISO 19115, EML, DublinCore, DataCite, DDI, and others. 

An overview of the web application is here: https://nceas.github.io/metacatui/

Some example repositories that utilize Metacat and MetacatUI for a data catalog:

- Arctic Data Center (https://arcticdata.io)
- KNB Data Repository (https://knb.ecoinformatics.org)
- ESS-DIVE Repository (https://ess-dive.lbl.gov/)

**Comment by pacian** [comment]
Hi @mbjones,

Thank you for your response and the information provided. I will look over the information you gave me. We are in search of a domain-agnostic metadata catalog solution. We have reviewed CKAN, Invenio, and Dataverse; however, based on the initial details provided, Metacat would fit our needs.  

Do you know of any Biomedical organization currently using Metacat?

Again, thank you so much for your help.

Regards,
Richard

**Comment by mbjones** [comment]
Happy to discuss it. Most repositories that I know using our software are either general purpose, cross-disciplinary groups, or groups focused on earth and environmental sciences. The Arctic Data Center, for example, holds data across all fields of science and engineering funded by the National Science Foundation in the Arctic. Some of the repositories hold a lot of biological data, including experimental work, and biodiversity data including some environmental genomics studies. I don't know of any explicitly biomedical organizations using it though. While we do have some features for private and sensitive data classification, depending on the specifics of your biomedical data, there might be some issues to consider. For example, we do not encrypt data at rest (nor do the other systems you mentioned AFAIK), which might be a consideration for highly sensitive HIPAA data, for example.

**Comment by pacian** [comment]
Great to hear back from you! 

We aim to create a metadata catalog using Metacat / MetacatUI without uploading any datasets to the repository. Instead, we plan to collect and store metadata for datasets that have already been uploaded to other domain-specific repositories such as EMBL. Researchers can easily search and discover relevant datasets without navigating multiple repositories.

In terms of compliance, we will ensure that our metadata catalog adheres to all relevant regulations, including GDPR and HIPAA requirements. I am located in Denmark, and we will ensure that our platform complies with any local data protection laws and regulations.

**Comment by yvanlebras** [comment]
Hi Richard @pacian As we, in French Biodiversity e-infrastructure, are working around using EML as a metadata "pivot format" in Biodiversity including genetics / genomics data, we are considering applying it to life sciences linked to biomed, in France and Europe. We have a metacat + metacatui data catalog where we are finalizing this year the qualitative criteria to populate it with all the french biodiversity oriented data. If relevant, don't hesitate to propose to discuss particularly about opportunity to use a combined approach with EML and metacat/metacatui to test things inline with ELIXIR infrastructure.
